pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -205,8 +205,8 @@ def __init__(self,
 
     def extract_data(self):
         super().extract_data()
-        self.metadata = self.raw_data_dict.get('exp_metadata', [{}])
-        if self.metadata is None:
+        self.metadata = self.raw_data_dict.get('exp_metadata', [])
+        if len(self.metadata) == 0:
             self.metadata = {}
```

Comment:
Change made by @Vintrix . Was because if exp_metadata is empty / there is no meta data, an empty list is returned, therefore it does not set it to the wanted value of {}.
I agree that a better fix would be to check why an empty list is returned by the load hdf

Reply:
I trust this works then. Its not nice, but I think we can leave it for now.